server.js

```diff
@@ -15,6 +15,61 @@ app.get("/", function (request, response) {
 
 // TODO add your routes and helper functions here
 
-const listener = app.listen(process.env.PORT, function () {
+app.get("/booking", function (req, res) {
+  res.send(bookings);
+});
+
+app.get("/booking/:id", function (req, res) {
+  const bookingId = +req.params.id;
+  const booking = bookings.find((booking) => booking.id === bookingId);
+  if (!booking) {
+    res.status(404).send("404 error");
+  } else {
+    res.send(booking);
+  }
+});
+
+app.post("/booking", function (req, res) {
+  const createBooking = req.body;
+  createBooking.id = bookings.length + 1;
```

Comment:
This will work currently, however you can't rely on the length of an array to give you a unique identifier in real applications; for instance, what if a booking is deleted then recreated? You now have `bookingId = 2` referring to different bookings at different times (therefore it is not a good identifier). Also if multiple bookings come in at the same time, you could end up with multiple `bookingId = 2` as they will all see the current length of the array could be `1` or something. A `uuid` is what you're looking for here.